khal/configwizard.py

```diff
@@ -185,6 +209,132 @@ def create_vdir(names=None):
     return [(name, path, 'calendar')]
 
 
+# Parsing and then dumping config naively could lose comments and formatting.
+# Since we don't need to modify existing fields, we can simply append our new
+# config to the end of the file.
+VDS_CONFIG_START = """\
+[general]
+status_path = "~/.local/share/vdirsyncer/status/"
+"""
+
+VDS_CONFIG_TEMPLATE = """
+[pair khal_pair_{pairno}]
+a = "khal_pair_{pairno}_local"
+b = "khal_pair_{pairno}_remote"
+collections = ["from a", "from b"]
+
+[storage khal_pair_{pairno}_local]
+type = "filesystem"
+path = {local_path}
+fileext = ".ics"
+
+[storage khal_pair_{pairno}_remote]
+type = "caldav"
+url = {url}
+username = {username}
+password = {password}
+"""
+
+
+def vdirsyncer_config_path():
+    """Find where vdirsyncer will look for it's config.
+
+    There may or may not already be a file at the returned path.
+    """
+    fname = environ.get('VDIRSYNCER_CONFIG', None)
+    if fname is None:
+        fname = normpath(expanduser('~/.vdirsyncer/config'))
+        if not exists(fname):
+            xdg_config_dir = environ.get('XDG_CONFIG_HOME',
+                                         normpath(expanduser('~/.config/')))
+            fname = join(xdg_config_dir, 'vdirsyncer/config')
+    return fname
```

Comment:
I wonder if it wouldn't be best to expose this functionality on vdirsyncer, rather than duplicate this here.

A bit out of-scope tho, so this is not a blocking comment.

Reply:
probably, if we are at it, it would also be really nice if it could print all synchronized collections (and their type).